content/ch-quantum-hardware/error-correction-repetition-code.ipynb

```diff
@@ -7031,7 +7031,7 @@
    "source": [
     "The resulting logical error probabilities are displayed in the following graph, whch uses a log scale used on the y axis. We would expect that the logical error probability decays exponentially with increasing $n$. If this is the case, it is a confirmation that the device is compatible with this basis test of quantum error correction. If not, it implies that the qubits and gates are not sufficiently reliable.\n",
     "\n",
-    "Fortunately, the results from IBM Q prototype devices typically do show the expected exponential decay. For the results below, we can see that small codes do represent an exception to this rule. Other deviations can also be expected, such as when the increasing the size of the code means uses a group of qubits with either exceptionally low or high noise."
+    "Fortunately, the results from IBM Q prototype devices typically do show the expected exponential decay. For the results below, we can see that small codes do represent an exception to this rule. Other deviations can also be expected, such as when an increase in the size of the code introduces a group of qubits with either exceptionally low or high noise."
```

Comment:
I am not sure if "introduce" is the right word here. I would rather say "could include".
But I want @quantumjim to check this.

Reply:
"means using" is what I originally intended. "introduces" is not so bad, but perhaps "could include" is better.

Reply:
@quantumjim changed it. Maybe "such as when xxx includes qubits that..." sounds more right than "such as when xxx could include qubits that..."? Either way, I think the sentence is probably clearer than before.